lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -973,6 +973,21 @@ class _PointerAdapter extends _BaseAdapter with _WheelEventListenerMixin {
         );
       _convertEventsToPointerData(data: pointerData, event: event, details: down);
       _callback(event, pointerData);
+
+      if (event.target == _viewTarget) {
+        // The event default is prevented so that the <flutter-view /> element doens't gain focus.
+        event.preventDefault();
+        // We give time to TextFields to move the focus to the right <input /> or <textarea />
+        // elements. When requestViewFocus is called it will make sure that a child of the
+        // <flutter-view /> is focused, otherwise it will focus the <flutter-view /> itself.
+        Timer(Duration.zero, () {
```

Comment:
Why do we need to wait here? Is it so the browser has time to render a real `input`, or something else?

Reply:
PTAL

Reply:
This makes sense, but at the end of the day it's flutter the one rendering inputs, I wish there was some event-driven way of doing this rather than "kick it to the next frame and hope it's there :P"